app/Http/Controllers/API/TorrentController.php

```diff
@@ -434,6 +436,7 @@ public function filter(Request $request): TorrentsResource|\Illuminate\Http\Json
                 ->when($request->filled('adult'), fn ($query) => $query->ofAdult($request->boolean('adult')))
                 ->when($request->filled('free'), fn ($query) => $query->ofFreeleech($request->free))
                 ->when($request->filled('doubleup'), fn ($query) => $query->doubleup())
+                ->when($request->filled('refundable'), fn ($query) => $query->where('refundable', $request->boolean('refundable')))
```

Comment:
Can you please move this filter to a scope in https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/app/Traits/TorrentFilter.php

Reply:
Looks like we already have a scope in that file actually. You can use it directly.

Reply:
@Roardom Updated as requested.